xml/ay_bigfile.xml

```diff
@@ -12993,32 +12993,23 @@ exit 0
   </sect1>
 
   <sect1 xml:id="CreateProfile.firewall">
-   <title>SUSE Firewall</title>
+   <title>Firewall Configuration</title>
 
    <para>
-    SUSE Firewall can be configured using the <literal>firewall</literal>
-    resource. All the properties in this resource are optional and all of them
-    are of type text (except the boolean
-    <literal>start_firewall</literal> and <literal>enable_firewall</literal> properties).
-   </para>
+     SuSEfirewall2 has been replaced by <literal>firewalld</literal> since
+     &slsa; 15. Profiles using SuSEfirewall2 properties will be translated
+     to <literal>firewalld</literal> but only some of them will be really
+     supported. Refer to <xref linkend="sec.security.firewall.firewalld"/>
+     if you want to know details about <literal>firewalld</literal>.
```

Comment:
As it is not `firewalld` documentation, I would add *about firewalld support*. Or just "refer to ... for further details.".